torch/__init__.py

```diff
@@ -270,22 +270,30 @@ def set_default_tensor_type(t):
 
 
 def set_default_dtype(d):
-    r"""Sets the default floating point dtype to :attr:`d`. This type will be
-    used as default floating point type for type inference in
-    :func:`torch.tensor`.
+    r"""Sets the default floating point dtype to :attr:`d`.
+    This type is also used:
+    1. As default floating point type for type inference in :func:`torch.tensor`.
+    2. To determine the default complex dtype. The default complex dtype is set to
+       ``torch.complex128`` if default floating point tensor type is ``torch.float64``,
+       else it's set to ``torch.complex64``
```

Comment:
nit: else -> otherwise.

Reply:
is it not true that this is the corresponding complex dtype for the float default?

Reply:
that's true